packages/core/test/integration/json.test.ts

```diff
@@ -73,6 +73,9 @@ describe('JSON Querying', () => {
 
   const vars = beforeAll2(async () => {
     class User extends Model<InferAttributes<User>, InferCreationAttributes<User>> {
+      @Attribute(DataTypes.INTEGER)
+      @PrimaryKey
```

Comment:
by the way technically you only need to add the declare field, the decorators are not necessary as the property will still be auto-added

Reply:
Ok, I've removed these now